### ISSUE TITLE
Add runAsUser e fsGroup securityContext configuration

### ISSUE THREAD

**Issue by tauonico** [TykTechnologies/tyk-helm-chart#178]
Fixed value in template for 'runAsUser' and 'fsGroup' cannot be validated against security context constraint on some environment.

Example:
```
securityContext:
        runAsUser: 1000
        fsGroup: 2000
```

To be:
```
securityContext:
        runAsUser: {{ .Values.securityContext.runAsUser }}
        fsGroup: {{ .Values.securityContext.fsGroup }}
```

**Comment by buraksekili** [comment]
Hi @tauonico, we have an open ticket (#220) for `securityContext` configuration - hoping to release it soon.

**Comment by buraksekili** [comment]
@tauonico we merged `securityContext` configuration #220 to the master. It will be part of the next release. Thanks!!
